src/routing/Api.ts

```diff
@@ -162,14 +163,17 @@ export async function info() {
 export async function routeWithAlternativeRoutes(requestId: number, args: RoutingArgs) {
     const request: RoutingRequest = {
         vehicle: args.vehicle || 'car',
-        elevation: false,
+        elevation: true,
         debug: false,
         instructions: true,
         locale: 'en',
         optimize: 'false',
-        points_encoded: true,
+        // todonow: apparently point decoding does not handle elevation yet
+        points_encoded: false,
```

Comment:
Right? Shall we fix this first?

Reply:
I just had to pass the is3D flag to a couple of methods, I think this is needed because we cannot tell from the encoded string whether it is 3D or not @karussell?

Reply:
Yes, the response does not contain this information. (Should we include this to make it easier to consume on the client side?)

Reply:
No, I think it is easy enough to keep track of the request's elevation flag here. So unless there have been complaints about this in the past no need to change the response format IMO.